### PR TITLE
Add prompt for grunt-modernizr

### LINKED ISSUE #107: Integrate grunt-modernizr
Should we integrate grunt-modernizr? Would be nice if we had a lean build based on what's actually used instead of shipping with the full-blown copy.

https://github.com/Modernizr/grunt-modernizr

### PR DESCRIPTION
**Pull request by passy**
Fix #107

This adds a prompt for `grunt-modernizr` and removes it completely, if the checkbox is unchecked (#124). I chose to directly overwrite the bower component for Modernizr, which seemed the easiest way. The grunt task is actually really well customizable, so if a detection doesn't work, you can still manually enable or disable certain checks or turn of the autodetection completely.

### CODE REVIEW

File: app/templates/Gruntfile.js [hunk 1]
```diff
@@ -330,6 +330,16 @@ module.exports = function (grunt) {
                 src: '{,*/}*.css'
             }
         },
+        modernizr: {
```
Comment (sindresorhus) [inline app/templates/Gruntfile.js:333]: `<% if (includeModernizr) { %>`

Comment (passy) [inline app/templates/Gruntfile.js:333]: Woops, thanks.

**Comment by sindresorhus** [comment]
WGTM

**Comment by sindresorhus** [comment]
:horse:

**Comment by passy** [comment]
:unicorn: (I'm sure it'll work one day.)

**Pull request merged**